tools/export/uvision/uvision.tmpl

```diff
@@ -394,7 +394,7 @@
               <MiscControls>{{asm_flags}}</MiscControls>
               <Define></Define>
               <Undefine></Undefine>
-              <IncludePath>{{include_paths}}</IncludePath>
+              <IncludePath></IncludePath>
```

Comment:
This actually reverts ebc9289d3042627b81b5cf03fc8caa3b7ccdafd9 and solves the problem @hug-dev was having by updating the path in targets/TARGET_ARM_SSG/TARGET_CM3DS_MPS2/device/TOOLCHAIN_ARM_STD/startup_MPS2.S ? The include is still there, where does assembler finds it? - not clear to me.